<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentalApp</class>
 <widget class="QWidget" name="RentalApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="mainPage">
    <widget class="QPushButton" name="extraBtn">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>279</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Дополнительные запросы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewBtn">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>340</y>
       <width>279</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отображение представления</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="clientBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Список клиентов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filmBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Список фильмов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rentBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Список прокатов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="expiredBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Список просрочки</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>681</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Таблицы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дополнительные запросы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="clientPage">
    <widget class="QTableView" name="clientTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>10</y>
       <width>301</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ФИО:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fio">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Номер телефона:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phone">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>День рождения:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bday">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Паспорт:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passport">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addClientBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="UpdateClientBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteClientBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="filmPage">
    <widget class="QTableView" name="filmTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1211</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>610</y>
       <width>195</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Возрастное ограничение:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>530</y>
       <width>98</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Цена в день:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>590</y>
       <width>153</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество дисков:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="genre">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>530</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="producer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="year">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>570</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="age">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>610</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="price">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>530</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="quantity">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>590</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>78</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>291</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Продюссер:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>530</y>
       <width>44</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Жанр:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>570</y>
       <width>33</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Год:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>500</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="makeRent_1">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string> В прокат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addFilmBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Добавить фильм</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="UpdateFilmBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Изменить фильм</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteFilmBtn">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Удалить фильм</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="rentPage">
    <widget class="QTableView" name="rentTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1211</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="clientSelection">
    <widget class="QTableView" name="selectionTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1211</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="days">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>600</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>600</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Дни:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="makeRent_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>600</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>В прокат</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="expiredPage">
    <widget class="QTableView" name="expiredTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1211</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="extraPage">
    <widget class="QTableView" name="extraTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>1211</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="queries">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="viewPage">
    <widget class="QTableView" name="viewTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>1211</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="display">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>exitBtn</zorder>
  <zorder>backBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
